docs/sidebar.js

```diff
@@ -73,7 +73,8 @@ module.exports = {
     "guides/login-session",
     "guides/secret-key-rotation",
     "guides/high-availability-ha",
-    "guides/docker"
+    "guides/docker",
+    "guides/kratos-video-tutorials"
```

Comment:
Wouldn't this make more sense in the "Further Reading" section?

Reply:
It is not very essential true, I just thought put it there because it directly related to the guides above.

Reply:
Yeah ok makes sense!

Reply:
As we discussed this changed to `.json` format - could you update the PR or make a new one?